React/Base/RCTTouchHandler.m

```diff
@@ -150,6 +150,11 @@ - (void)_updateReactTouchAtIndex:(NSInteger)touchIndex
   reactTouch[@"locationX"] = @(touchViewLocation.x);
   reactTouch[@"locationY"] = @(touchViewLocation.y);
   reactTouch[@"timestamp"] =  @(nativeTouch.timestamp * 1000); // in ms, for JS
+  
+  if([UIDevice currentDevice].systemVersion.floatValue >= 9) {
+    reactTouch[@"force"] = @(nativeTouch.force);
+    reactTouch[@"maxForce"] =  @(nativeTouch.maximumPossibleForce);
```

Comment:
I'd rather avoid another iOSisms in the attribute naming, those two properties can and should be cross-platform on android referring to `MotionEvent#getPressure` (max is fixed to 1.0)